src/Api/RequestAttributesExtractor.php

```diff
@@ -9,44 +9,20 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Action;
+namespace ApiPlatform\Core\Api;
 
-use ApiPlatform\Core\DataProvider\ItemDataProviderInterface;
 use ApiPlatform\Core\Exception\RuntimeException;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 
 /**
- * Checks if the request is properly configured.
+ * Extract useful data form a request managed by API Platform.
```

Comment:
Extracts ... from ... ~~managed by API Platform~~ (this part is just confusing...)

But I'm not sure "useful data" tells us anything useful :laughing: